### PR TITLE
Add modification date to parties response

### LINKED ISSUE #39: Add register date
It would be useful to have the date registered recorded and included in the `GET /parties` response.

This would make it easier for downstream scripts to add tokens for all parties registered more recently than a certain date.

## current output
```json5
[
  {
    "party_id": "01...ef",
    "twitter_handle": "___",
    "twitter_user_id": 123456789
  },
  // ...
]
```

## desired output
```json5
[
  {
    "party_id": "01...ef",
    "created": 1618474460,  // epoch seconds
    "last_modified": 1618474499,  // edit: let's have this too, also epoch seconds
    "twitter_handle": "___",
    "twitter_user_id": 123456789
  },
  // ...
]
```

### PR DESCRIPTION
**Pull request by fkondej**
Closes #39.
New response format:
```json
[
  {
    "party_id": "75b5b97f272342432e0cf56a3432424678a6d24f1675e95486ee2a6d31e",
    "twitter_handle": "twitter_account",
    "twitter_user_id": 285745133,
    "last_modified": 1618395417,
    "created": 1618395417
  }
]
```

### CODE REVIEW

**Comment by ashleyvega** [comment]
Strictly speaking, I asked for "created", not "last_modified". I'm interested in the _first_ time someone registered.

But this raises yet another question: If a party re-registers (same twitter ID, new partyID), do they get more tokens? are they registering for a new competition or re-registering for an old one? :-|

**Comment by ashleyvega** [comment]
Also, I asked for epoch seconds (integer), not a datetime that a script has to parse.

**Comment by ashleyvega** [comment]
Just in case needed, let's have _both_ `created` and `last_modified`, please (both epoch integers).

**Comment by fkondej** [comment]
> Strictly speaking, I asked for "created", not "last_modified". I'm interested in the _first_ time someone registered.
> 
> But this raises yet another question: If a party re-registers (same twitter ID, new partyID), do they get more tokens? are they registering for a new competition or re-registering for an old one? :-|

Good question. I am not going to address this here. It is better to move it to #28.

File: src/services/smv_storage.py [hunk 1]
```diff
@@ -31,6 +31,18 @@ def get_parties(self):
                 "party_id": item["pub_key"],
                 "twitter_handle": item["twitter_handle"],
                 "twitter_user_id": item["twitter_user_id"],
+                "last_modified": int(
+                    item["last_modified"]
+                    .replace(tzinfo=timezone.utc)
+                    .timestamp()
+                ),
+                # created might be missing in DB - use "last_modified" then
+                # TODO: remove usage of "last_modified" once no longer neede
```
Comment (ashleyvega) [inline src/services/smv_storage.py:40]: Now that the db has been nuked, can we remove the TODO and replace `item.get("created", item["last_modified"])` with `item["created"]` ?

Comment (fkondej) [reply src/services/smv_storage.py:40]: yes, we can :)

**Review by ashleyvega — APPROVED** [review]
LGTM

**Pull request merged**